sdk/servicebus/service-bus/src/core/batchingReceiver.ts

```diff
@@ -161,6 +161,30 @@ export class BatchingReceiver extends MessageReceiver {
         }
       };
 
+      // Use new message wait timer only in peekLock mode
+      if (this.receiveMode === ReceiveMode.peekLock) {
```

Comment:
Don't we have to do a similar code change in the messageSession.ts file in order to fix `receiveMessages()` when using session enabled queues/subscriptions?

Reply:
Ah, yes - I thought I specifically added it in.. updated